package.json

```diff
@@ -235,7 +235,7 @@
                 },
                 {
                     "command": "staticWebApps.browse",
-                    "when": "view == staticWebApps && viewItem =~ /azureStatic(WebApp|Environment)$/",
+                    "when": "view == staticWebApps && viewItem == azureStaticEnvironment",
```

Comment:
Idk browse seems super important to me, so I worry about making it less discoverable. I think it makes sense what you did for the command palette, but can we keep the context menu on both?

Reply:
I'm kind of mixed on this because I think guiding the user to think that SWA === production environment could lead to some confusion.  The SWA itself doesn't have a website or host, everything in the portal forces you to select an environment before you can configure something.

![image](https://user-images.githubusercontent.com/5290572/88426704-7d611100-cda6-11ea-8ad3-227c11b0641c.png)

You can see that browse is on the production environment here as well  That being said, this is their overview pane:

![image](https://user-images.githubusercontent.com/5290572/88426790-9e296680-cda6-11ea-8ce2-ae9dc9899c53.png)

But with all that said, I'm fine with having browse on both, just some considerations.